code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -536,3 +537,78 @@ get_true_breath_pressure(pp) --> gas_pp = pp/breath_pp*total_moles()
 
 		return TRUE
 	return FALSE
+
+
+/**
+ * Condenses the gases in the gas mixture into reagents in a areagent container
+ *
+ * Arguments:
+ * - [location][/atom]: The location the condensation is happening at
+ * - [reagents][/datum/reagents]: The reagent holder to output the condensed gases to
+ * - delta_time: The amount of time the condensation is happening over
+ */
+/datum/gas_mixture/proc/condense_gas_to(atom/location, datum/reagents/reagents, delta_time)
+	if(delta_time <= 0)
+		return
```

Comment:
Because, unfortunately, the delta_time passed by the condenser cannot be trusted to be positive. It basically exists in case some admeme vv's a condensers condensation efficiency into the negatives.